packages/genesys-spark-components/src/components/stable/gux-list/gux-list.tsx

```diff
@@ -84,14 +84,9 @@ export class GuxList {
     last(this.root, validFocusableItems);
   }
 
-  private renderFocusTarget(): JSX.Element {
-    return (<span tabindex="-1" aria-hidden="true"></span>) as JSX.Element;
-  }
-
   render(): JSX.Element {
     return (
-      <Host role="list">
-        {this.renderFocusTarget()}
```

Comment:
There is an issue with having this focus target element. Both of the following cause accessibility issues
1. Having a focusable element with a role of `aria-hidden` is a violation
2. If you remove the `aria-hidden`, having an element with an invalid role as a descendent of the `list` is a violation.
I have removed this focus target span and instead put a tab index of `-1` on the Host.